library/haskell

```diff
@@ -2,10 +2,15 @@ Maintainers: Peter Salvatore <peter@psftw.com> (@psftw),
              Christopher Biscardi <chris@christopherbiscardi.com> (@ChristopherBiscardi)
 GitRepo: https://github.com/haskell/docker-haskell
 
-Tags: 8.8.1, 8.8, 8, latest
+Tags: 8.8.3, 8.8, 8, latest
 GitCommit: aa517c24c51bbd54da5805517acd12120a9340ab
```

Comment:
This commit should be updated to `083c587a8f5d707450683a846ebfd4a405f21719` so that `8.8.1` doesn't just get tagged as `8.8.3` (and on #7554 as well).

Reply:
yikes, thank you for catching. It should be fixed in both PRs now.